src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -963,6 +963,9 @@ impl<'a> BinaryBuffer<'a> {
                     MacroRef::new(macro_address, macro_ref),
                     bitmap_bits,
                     matched_eexp_bytes,
+                    // There is no length prefix, so we re-use the bitmap_offset as the first position
+                    // beyond the opcode and address subfields.
+                    bitmap_offset as u8,
```

Comment:
🗺️ This is in the parser for e-expressions with combined opcode+address bytes. There is never a length prefix.